cmd/instances.go

```diff
@@ -437,6 +437,36 @@ var oauthClientInstanceCmd = &cobra.Command{
 	},
 }
 
+var updateAllCmd = &cobra.Command{
+	Use:   "update-all [slugs...]",
+	Short: "Starts the updates for all instances.",
+	Long: `Starts the updates for all instances. The slugs arguments can be used
+to select which applications should be updated.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		c := newAdminClient()
+		return c.Updates(&client.UpdatesOptions{
+			Slugs: args,
+		})
+	},
+}
+
+var updateCmd = &cobra.Command{
+	Use:   "update [domain] [slugs...]",
```

Comment:
What about the `cozy-stack apps update` command? Does this command replace it? Or are they are complementary?